homeassistant/components/axis.py

```diff
@@ -0,0 +1,347 @@
+"""
+Support for Axis devices.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/axis/
+"""
+
+import json
+import logging
+import os
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.helpers.event import track_point_in_utc_time
+from homeassistant.util.dt import utcnow
+
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers import discovery
+
+from homeassistant.const import (ATTR_LOCATION, ATTR_TRIPPED,
+                                 CONF_HOST, CONF_INCLUDE, CONF_NAME,
+                                 CONF_PASSWORD, CONF_SCAN_INTERVAL,
+                                 CONF_USERNAME, EVENT_HOMEASSISTANT_STOP)
+from homeassistant.helpers.entity import Entity
+from homeassistant.loader import get_component
+
+
+# REQUIREMENTS = ['axis==5']
```

Comment:
I'm the author of it, but I haven't gotten it to pipy. I will remove that comment, it was mostly to speed up testing things